regression-test/suites/partition_p0/test_partition_operation.groovy

```diff
@@ -37,6 +37,7 @@ suite("test_partition_operation", "p1") {
             AGGREGATE KEY(k1,k2,k3,k4,k5,k6) 
             ${partitionInfo} 
             ${distributionInfo}
+            BUCKETS 8
```

Comment:
why change all buckets to 8 ???

Reply:
To avoid problems like that not appear in test.

Reply:
line 72 already has 'BUCKETS 13'